src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        File file = new File(filePath);
```

Comment:
It's recommended to use try-with-resources to ensure that the BufferedReader is closed properly. However, you've already done this correctly, so no changes are needed here.